pkgs/development/tools/faq/default.nix

```
@@ -0,0 +1,42 @@
+{ buildGoModule, pkgs, lib, fetchFromGitHub }:
+
+buildGoModule rec
+{
```

That's very unusual formatting. How about:

```suggestion
buildGoModule rec {
```